### ISSUE TITLE
After conversion songs missing in Apple Music

### ISSUE THREAD

**Issue by Harbersbram** [therealmarius/Spotify-2-AppleMusic#34]
Hey, thank you for the great work. I have some large playlists I want to move over, about 1700 tracks. The first time I executed the script, I got a 99% succes rate, saying about 1690 tracks had been successfully transferred. When opening the playlist in Apple music, only 1280 tracks have been transferred, leaving about 400 tracks completely absent.

The beginning and end of the playlist remain the same, meaning somewhere in the middle the tracks have disappeared. I then split the Spotify playlist in half, to try to do it in parts. Once again, when it's finished, about 200 tracks are unaccounted for. 

Any idea how to fix this issue?

**Comment by therealmarius** [comment]
Hey!

I understand your issue. Sometimes Apple Music takes some time to reflect the song uploaded into the playlist. If the script told you that they were successfully uploaded, it means that Apple Music sent a response code meaning that they have a understood that you wanted to add *that* song to your playlist.

They should appear after some time.

Please keep us posted.

Best,
therealmarius

**Comment by therealmarius** [comment]
Hi,

I may have found a solution: if you have a Mac try updating your cloud library. To do so, go over to File > Library > Update Cloud Library

Hope it fixes your issue.

Best,
therealmarius

**Comment by therealmarius** [comment]
Hi (again),

After doing some tests for a new version of the script, I still have the same issue as you: some tracks are missing...
I tried adding more delay between each track, but it doesn't change anything. Also I noticed that it's always the same. tracks that doesn't show up... Maybe it's a backend thing. Strange thing, I can add them using the web interface that basically send the same requests as the script.

Sorry it doesn't really help you with your issue.

Best,
therealmarius
